### PR TITLE
tweak(NuclearBomb Keg): Give the nuclear bomb keg a hit box

### PR DESCRIPTION
**Pull request by Brandon-Huu**
## About the PR

Gave the nuclear bomb keg a hit box.

## Why / Balance

Less metagaming
It should have a hitbox

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Before**
[Screencast from 07-27-2024 01:04:05 PM.webm](https://github.com/user-attachments/assets/94190e41-8dcb-44f7-ab13-2361d350fa26)

**After**
[Screencast from 07-27-2024 01:04:14 PM.webm](https://github.com/user-attachments/assets/f9541a9d-9a86-4a12-9bfd-81505538a125)

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Brandon-Huu** [comment]
bugged

**Comment by Brandon-Huu** [comment]
No more bug

**Comment by Brandon-Huu** [comment]
Trust me this time

**Comment by Brandon-Huu** [comment]
another bug?
How would I fix this?
Should I fix this?
The bug:
[Screencast from 07-27-2024 01:24:41 PM.webm](https://github.com/user-attachments/assets/676cd929-7aaf-485f-8953-192a25268d51)

No it doesn't get stuck if you throw stuff at it:
[Screencast from 07-27-2024 01:25:39 PM.webm](https://github.com/user-attachments/assets/81b62266-b5be-47ac-a463-a146277c4215)

File: Resources/Prototypes/Entities/Structures/Machines/nuke.yml [hunk 1]
```diff
@@ -161,11 +163,11 @@
         shape:
           !type:PhysShapeCircle
           radius: 0.45
-        density: 80
+        density: 255 # Has "bluespace technology" to make it lighter, whatever that means. Don't mind the fact that this is lighter then a high capacity fuel tank.
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Entities/Structures/Machines/nuke.yml:166]: Unnecessary lore

**Review by VasilisThePikachu — CHANGES_REQUESTED** [review]

**Review by VasilisThePikachu — APPROVED** [review]

**Pull request merged**